src/fft-api/common/constants.ts

```diff
@@ -0,0 +1 @@
+export const MAX_ARRAY_SIZE = 500;
```

Comment:
Haben wir die 500 überall?

Reply:
Kurzer check von `maxItems` in der API Spec - es gibt alles
1, 3, 5, 10, 20, 23, 50, 100, 200, 500